scripts/pgs/LDpred2/ldpred2.R

```diff
@@ -152,6 +153,20 @@ if (genoImputeZero) {
 cat('\n### Reading LD reference meta-file from ', fileMetaLD, '\n')
 map_ldref <- readRDS(fileMetaLD)
 
+# rename pos column in map_ldref if another genomic build is assumed:
```

Comment:
Just a suggestion...
Move the argument in `stop(...)` up so that the check is performed prior to reading any data `if (!parsed$genomic_build %in% c(...)) stop(...)`. Then I think you can skip this `if (parsed... != ''hg19')` since this is the default.

Reply:
Fixed.